src/main/java/inf112/isolasjonsteamet/roborally/gui/MapRendererWidget.java

```diff
@@ -31,16 +30,19 @@ public MapRendererWidget(ClientBoard board, int tileSize) {
 		sizeChanged();
 	}
 
+	/**
+	 * Draw method.
+	 */
 	@Override
 	public void draw(Batch batch, float parentAlpha) {
 		super.draw(batch, parentAlpha);
 		// The map renderer does not behave well together with scene2d's sprite batch,
 		// so we need to stop the current one, and restart it after we have rendered the map.
 		batch.end();
-
```

Comment:
Don't remove the newlines here. They're useful to indicate which batch is active where